### DISCUSSION TITLE
Osu!mania ss rank

### DISCUSSION THREAD

**Discussion by ghost** [ppy/osu#30695] [Q&A] [answered]
I think difficulties of ss in osu!stable and osu!lazer are way much different. Isn't it better to make criterias the same? I don't get that the ss criteria in lazer is made different from osu!stable in the first place. I think at least ss scores in osu!stable have to be recalculated and either be changed to s or not in the osu!lazer mode as well as in non osu!lazer mode (in the website).

**Comment by peppy** [comment, marked as answer]
This will be fixed in the next release (https://github.com/ppy/osu/pull/29640)
